language_reference_manual.md

```diff
@@ -5,6 +5,76 @@
 ## Types
 
 ## Operators 
+### Arithmetic Operators
+The binary arithmetic operators are +, -, *, /, % .
+Integer division truncates any fractional part.
+The binary + and - operators have the same precedence, which is lower than the
+precedence of *, /, % . Arithmetic operators associate left to right.
+
+### Grammar
+
+```
+expr:
+expr PLUS expr  }
+| expr MINUS expr }
+| expr TIMES expr }
+| expr DIVIDE expr }
+| expr MODULO expr }
+```
+
+### Example Syntax
+
+```
+int x = 5 * (16 - 6);
+```
+
+### Assignment Operators
+The operator = assigns an expression to a variable. It assigns value on right of operator to the left.
+
+### Example Syntax
+
+```
+int x = 5;
+```
+
+### Comaparision Operators
+The relational operators are <, <=, >, >= . They all have the same precedence. 
+
```

Comment:
I think you're missing equivalence operators, logical operators, and special functionality (e.g. pipe `|` operator). Please see #5 for more details about what I was thinking for these.